src/movies.js

```diff
@@ -34,14 +34,40 @@ function ratesAverage(movies){
       }
       });
     console.log(reducer);  
-    var average = reducer.reduce(function (accumulator, cucurrentValue) {
+    var sum = reducer.reduce(function (accumulator, cucurrentValue) {
       return (accumulator + cucurrentValue)
-      }, 0 );
-    return Math.round((average/reducer.length)*100)/100
+      });
+    return Math.round((sum/reducer.length)*100)/100
   }
 }
   console.log(ratesAverage(movies));
 // Iteration 4: Drama movies - Get the average of Drama Movies
```

Comment:
Vous pouviez utiliser la function de l'itération 3, ça vous aurez simplifié le code : 
function dramaMoviesRate(movies) {
	const dramaMovies = movies.filter(movie => movie.genre.includes('Drama'))
	return (ratesAverage(dramaMovies));
}